lib/data_cycle/schedule_course_updates.rb

```diff
@@ -3,17 +3,20 @@
 require_dependency "#{Rails.root}/lib/data_cycle/batch_update_logging"
 require_dependency "#{Rails.root}/lib/data_cycle/course_queue_sorting"
 require_dependency "#{Rails.root}/app/workers/course_data_update_worker"
+require_dependency "#{Rails.root}/app/workers/update_wikidata_stats_worker"
 
 # Puts courses into sidekiq queues for data updates
 class ScheduleCourseUpdates
   include BatchUpdateLogging
   include CourseQueueSorting
 
   def initialize
+    @updated_courses = []
     setup_logger
     return if updates_paused?
 
     run_update
+    wikidata_stats_updates
```

Comment:
This strategy does not solve the problem of getting the wikidata updates in sync with the other course data updates. It will add all the CoursesDataUpdateWorker jobs to the sidekiq queue, and then add all the UpdateWikidataStatsWorker jobs to the queue, but then either all the first set of jobs will need to complete before the first wikidata job gets processed, or if there are multiple processes taking jobs from the queue, you could still end up with cases where the wikidata job runs before the other job for that course has completed.

I suggest, instead, that we start by including the wikidata update as *part of* the CourseDataUpdate for each course. You can do this while still keeping the worker framework by calling `UpdateWikidataStatsWorker.new.perform(course_id)`. Calling `perform` directly means it won't be enqueued, it will just execute that job inline. (This way, we can also take advantage of the logging framework and see the effects of this change in the course update logs in terms of how much time a given course takes to do this new stats processing.)